mobile-center-crashes/android/src/main/java/com/microsoft/azure/mobile/react/crashes/RNCrashesModule.java

```diff
@@ -114,12 +111,58 @@ public void run() {
     }
 
     @ReactMethod
-    public void crashUserResponse(boolean send, ReadableMap attachments, Promise promise) {
-        int response = send ? Crashes.SEND : Crashes.DONT_SEND;
-        if (mCrashListener != null) {
-            mCrashListener.setAttachments(attachments);
+    public void notifyWithUserConfirmation(int userConfirmation) {
```

Comment:
Can we rename this method to `notifyUserConfirmation` to keep it consistent with RN Java implementation? We try to keep JS and Java API consistent in RN SDK in most places.

Reply:
This calls an iOS method so we can't really mess with iOS conventions here I guess.

Reply:
ok, make sense 👍